persistence/relational-jdbc/src/main/java/org/apache/polaris/persistence/relational/jdbc/JdbcMetaStoreManagerFactory.java

```diff
@@ -96,14 +96,17 @@ private void initializeForRealm(
     // Materialize realmId so that background tasks that don't have an active
     // RealmContext (request-scoped bean) can still create a JdbcBasePersistenceImpl
     String realmId = realmContext.getRealmIdentifier();
+    // determine schemaVersion once per realm
+    final int schemaVersion = JdbcBasePersistenceImpl.loadSchemaVersion(datasourceOperations);
```

Comment:
Couldn't this be once per JVM?

Reply:
What I meant is that there is an ML thread about schema setup that will likely revisit this again: https://lists.apache.org/thread/6c7lzgy3675jrrtbrrxsm8l0xd1fo1mj

But I'm OK to merge this PR as is.

Reply:
Yea - that thread looks orthogonal IMO.